content/00.build/30.zksync-sso/50.getting-started.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Getting Started
+description: Learn how to integrate ZKsync SSO into your app.
+---
+
+Integrating ZKsync SSO into your app requires minimal amount of changes.
+::callout{icon="i-heroicons-light-bulb"}
+ZKsync SSO can be used standalone or added as one of the options in the onboarding library of your choice.
+::
+
+## Quick Start
+
+Just 2 steps:
+
+1. Install the ZKsync SSO SDK package.
+
+    ::code-group
+
+    ```bash [npm]
+    npm i zksync-sso
+    ```
+
+    ```bash [yarn]
+    yarn add zksync-sso
+    ```
+
+    ```bash [pnpm]
+    pnpm add zksync-sso
+    ```
+
+    ```bash [bun]
+    bun add zksync-sso
+    ```
+
+    ::
+
+1. Add ZKsync SSO connector to your app.
+
+    ::code-group
+
+    ```ts [wagmi]
+    import { zksync } from "viem/chains";
```

Comment:
Should we tell people to use `zksyncSepoliaTestnet` instead of mainnet? I'm worried people will copy and paste this and be confused why it's not working